x-pack/plugins/security_solution/server/lib/detection_engine/routes/index/signals_mapping.json

```diff
@@ -257,6 +257,11 @@
           "original_time": {
             "type": "date"
           },
+          "original_signal": {
+            "type": "object",
+            "dynamic": false,
+            "enabled": false
+          },
```

Comment:
NOTE: This will not allow indexing and searching of the signal but it will allow "conflicts" meaning if a customer has different mappings of "signal" then this will mix between the two correctly but the downside is the search ability of the original signal is not there.

You can search for other things and then see your original_signal in timeline still though.